### ISSUE TITLE
Postgres 12 - Array Editor does not work properly

### ISSUE THREAD

**Issue by curala70** [dbeaver/dbeaver#10123]
Refers to this issue: #403 

#### System information: 
- Windows 10 Pro (Version 10.0.19041 Build 19041)
- DBeaver version 7.2.2

#### Connection specification:
- PostgreSQL 9.6.12 and PostgreSQL 12.2.
- PostgreSQL JDBC Driver

#### Describe the problem you're observing:
- for new arrays everything works fine
- for existing arrays a double-click does not let me edit values
- for existing arrays I can only add elements with NULL-values or remove elements

#### Steps to reproduce, if exist:
- Have a table.
- Create new column with, for example, _varchar or _int2 type.
- add a new data-array with arbitrary values (works fine) and persist it
- reload the table (F5)
- try to edit some values of the just created array (does not work)
- try to add new elements to the array
- try to remove elements from the array
- try to persist the changes (results in the Errors mentioned below)

#### Include any warning/errors/backtraces from the logs
```
Error creating struct
  Unable to find server array type for provided name _varchar.

Error creating struct
  Unable to find server array type for provided name _int2.
```

Another "wish" from my side, if someone will work on this soon:

- Please make sure, I can handle NULL values inside arrays
- This can be done with an additional option (set NULL-value) on the right-click action
- Similar to right click on simple varchar or text data cells

Thanks in advance for your work on this.
For me, DBeaver is the best Open Source solution that wipes out all the disadvantages of pgAdmin4.

**Comment by kseniiaguzeeva** [comment]
Thank you for the report. I have an error in Error log only when I delete the data from array cell (brackets remain in it).
But the changes are saved. Also adding, editing array values work for new and existed rows in 7.2.3 DBeaver PostgreSQL 12.
![dbeaver_wySXbxj71X](https://user-images.githubusercontent.com/51405061/96704335-a50f4780-139c-11eb-8e64-e8cda6e5e776.png)

Is it possible to send DDL example to reproduce other cases?
Could you check 7.2.3 version https://dbeaver.io/download/?

**Comment by curala70** [comment]
I checked the new version 7.2.3.
I also created a new column (_varchar) for testing.

Still the same issues are coming up as mentioned already.

I made a video for you. Will send it per mail.

**Comment by serge-rider** [comment]
This bug was recently fixed as a part of #9964 .
Fix will be included in 7.2.4
